keps/sig-node/4191-split-image-filesystem/README.md

```diff
@@ -0,0 +1,891 @@
+- How does an n-3 kubelet or kube-proxy without this feature available behave when this feature is used?
+- How does an n-1 kube-controller-manager or kube-scheduler without this feature available behave when this feature is used?
+- Will any other components on the node change? For example, changes to CSI,
+  CRI or CNI may require updating that component before the kubelet.
+-->
+
+## Production Readiness Review Questionnaire
+
+<!--
+
+Production readiness reviews are intended to ensure that features merging into
+Kubernetes are observable, scalable and supportable; can be safely operated in
+production environments, and can be disabled or rolled back in the event they
+cause increased failures in production. See more in the PRR KEP at
+https://git.k8s.io/enhancements/keps/sig-architecture/1194-prod-readiness.
+
+The production readiness review questionnaire must be completed and approved
+for the KEP to move to `implementable` status and be included in the release.
+
+In some cases, the questions below should also have answers in `kep.yaml`. This
+is to enable automation to verify the presence of the review, and to reduce review
+burden and latency.
+
+The KEP must have a approver from the
+[`prod-readiness-approvers`](http://git.k8s.io/enhancements/OWNERS_ALIASES)
+team. Please reach out on the
+[#prod-readiness](https://kubernetes.slack.com/archives/CPNHUMN74) channel if
+you need any help or guidance.
+-->
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+If a container runtime is configured to store these separately, there is no really good way to roll these changes back.
+We will include a feature gate for best practices.
+
+- [x] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: KubeletSeparateDiskGC
+  - Components depending on the feature gate: Kubelet
+- [x] Other
+  - Describe the mechanism:
+  - Will enabling / disabling the feature require downtime of the control
+    plane? Yes.
+  - Will enabling / disabling the feature require downtime or reprovisioning
+    of a node? No.
+
+The best way to roll this change back:
+
+1. Configure your container runtime to not split the image filesystem.
+2. Restart the container runtime.
+3. Restart Kubelet with feature flag off.
+
+###### Does enabling the feature change any default behavior?
+
+This feature will only change default behavior if othe container runtime is set up to split
+the image filesystem.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+If a container runtime enables split disk, then it will not be possible to roll back these changes.
+Rolling back can be done by restarting the container runtime with split disk turned off (by changing the container runtime configuration).
```

Comment:
I think this section should largely focus on kubelet's behavior. as in: if the container runtime messes up: how will kubelet react? if container runtime correctly enables, but the admin tries to configure kubelet without feature gate: how will it react?

Reply:
updated. PTAL